.env.example

```diff
@@ -30,37 +41,41 @@ SPRINT_SIZE_BLOCK_NUMBER=0 # comma separated values defining the block heights o
 HEIMDALL_REPO="https://github.com/maticnetwork/heimdall.git" # repo of heimdall project
-HEIMDALL_BRANCH=develop# heimdall branch defining the version to be used in the devnet
+HEIMDALL_BRANCH=develop # heimdall branch defining the version to be used in the devnet
 CONTRACTS_REPO="https://github.com/maticnetwork/contracts.git" # repo of contracts project
 CONTRACTS_BRANCH=mardizzone/node-16 # contracts branch defining the version to be used in the devnet. To use node v16.x, please stick with it
 GENESIS_CONTRACTS_REPO="https://github.com/maticnetwork/genesis-contracts.git" # repo of genesis-contracts project
 GENESIS_CONTRACTS_BRANCH=master# Genesis contracts branch defining the version to be used in the devnet
 MATIC_CLI_REPO="https://github.com/maticnetwork/matic-cli.git" # repo of matic-cli to run a specific version remotely using express-cli
-MATIC_CLI_BRANCH=master# matic-cli branch used on the remote machines to start the environment
-DEVNET_BOR_USERS=ubuntu,ubuntu,ubuntu#users' names of VMs for all the nodes (comma separated). Its length must be equal to "TF_VAR_VALIDATOR_COUNT + TF_VAR_SENTRY_COUNT"
+MATIC_CLI_BRANCH=master # matic-cli branch used on the remote machines to start the environment
+DEVNET_BOR_USERS=ubuntu,ubuntu,ubuntu #users' names of VMs for all the nodes (comma separated). Its length must be equal to "TF_VAR_BOR_VALIDATOR_COUNT + TF_VAR_BOR_SENTRY_COUNT + TF_VAR_BOR_ARCHIVE_COUNT"
+DEVNET_ERIGON_USERS=ubuntu #users' names of VMs for all the nodes (comma separated). Its length must be equal to "TF_VAR_ERIGON_VALIDATOR_COUNT + TF_VAR_ERIGON_SENTRY_COUNT + TF_VAR_ERIGON_ARCHIVE_COUNT"
 BOR_DOCKER_BUILD_CONTEXT="https://github.com/maticnetwork/bor.git#c5569e4da9ebe0ce4e63aec571966c71234f7cfc" # todo change to develop once https://polygon.atlassian.net/browse/POS-979 is solved (docker build context for bor. Used in docker setup (TF_VAR_DOCKERIZED=yes))
 HEIMDALL_DOCKER_BUILD_CONTEXT="https://github.com/maticnetwork/heimdall.git#develop" # docker build context for heimdall. Used in docker setup (TF_VAR_DOCKERIZED=yes)
-VERBOSE=true# if set to true will print logs also from remote machines
-DD_API_KEY=DATADOG_API_KEY# Datadog API key
+VERBOSE=true # if set to true will print logs also from remote machines
+DD_API_KEY=DATADOG_API_KEY # Datadog API key
 
-# Bor and Heimdall Snapshot related variables (when running mainnet/testnet node)
+# Bor, Erigon and Heimdall Snapshot related variables (when running mainnet/testnet node)
 BOR_SNAPSHOT_URL=https://snapshot-download.polygon.technology/bor-mumbai-fullnode-2023-03-18.tar.zst # URL for bor data snapshot (refer: https://snapshot.polygon.technology/)
 HEIMDALL_SNAPSHOT_URL=https://snapshot-download.polygon.technology/heimdall-mumbai-fullnode-2023-03-18.tar.zst # URL for heimdall data snapshot (refer: https://snapshot.polygon.technology/)
+ERIGON_SNAPSHOT_URL= # URL for Erigon data snapshot (refer: https://snapshot.polygon.technology/)
```

Comment:
Maybe you can put this as default  https://snapshot-download.polygon.technology/erigon-mumbai-archive-2023-05-24.tar.zst